src/WebSocketServer/WebSocketClient.cs

```diff
@@ -0,0 +1,86 @@
+using Neo.Json;
+using System;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Neo.Plugins
+{
+    internal class WebSocketClient : IDisposable, IEquatable<WebSocketClient>
+    {
+        public WebSocket? Socket { get; init; }
```

Comment:
We don't need this with ?, only is needed because you want a T type that we don't need

Reply:
I don't understand. Referring to `T` type; we do need it! This a `BASE` CLASS. NOT DOING SO WILL LIMIT AND FLAW EXPANDABLE OF AND POINT OF THIS LIBRARY. If you want to keep limiting the `api` we can do that. Just look at `RpcServer` is outdated and cannot be expanded or updated easily. Which is why no one touches it and it in the state it's in now.